app/forms/select_mobile_number_form.rb

```diff
@@ -0,0 +1,32 @@
+class SelectMobileNumberForm
+  def self.extract_attributes(claim, mobile_check:)
+    new(claim, mobile_check).extract_attributes
+  end
+
+  def initialize(claim, mobile_check)
+    @claim = claim
+    @mobile_check = mobile_check
+  end
+
+  def extract_attributes
+    if @mobile_check == "use"
+      {
+        mobile_number: @claim.teacher_id_user_info["phone_number"],
+        provide_mobile_number: true,
+        mobile_check: @mobile_check
+      }
+    elsif @mobile_check == "alternative"
+      {
+        mobile_number: nil,
+        provide_mobile_number: nil,
+        mobile_check: @mobile_check
+      }
+    elsif @mobile_check == "declined"
+      {
+        mobile_number: nil,
+        provide_mobile_number: false,
+        mobile_check: @mobile_check
+      }
+    end
```

Comment:
Since all 3 ifs are for `@mobile_check` case-statement would work well here I think.

Reply:
Nice catch. Done 👍